ntoskrnl/mm/pagefile.c

```diff
@@ -329,8 +330,9 @@ MmAllocSwapPage(VOID)
             }
             MiUsedSwapPages++;
             MiFreeSwapPages--;
-            KeReleaseGuardedMutex(&MmPageFileCreationLock);
+            UpdateTotalCommittedPages();
```

Comment:
These two calls were good outside of the lock, like you had them before. That is, assuming we're making the updates interlocked.

Reply:
Done